save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPluginConfig.kt

```diff
@@ -101,9 +112,9 @@ data class WarnPluginConfig(
     companion object {
         /**
          * Default regex for expected warnings in test resources, e.g.
-         * `// ;warn:2:4: Class name in incorrect case`
+         * `// ;warn:ClassNameTest.java:2:4: Class name in incorrect case`
          */
-        internal val defaultInputPattern = Regex(";warn:(\\d+):(\\d+): (.+)")
+        internal val defaultInputPattern = Regex(";warn:(.+):(\\d+):(\\d+): (.+)")
```

Comment:
I think it's not required in expected warnings, because you'll need to repeat file name multiple time *inside the same file*. It makes sense to parse file name from tool's *output*, but not for input.